src/declaration.h

```diff
@@ -580,10 +580,10 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
-#ifdef IN_GCC
+#ifdef IN_GCC // %%
```

Comment:
@yebblies - We use v_argptr to stash the ``_argptr`` var, then generate the code in the backend.  (va_start and va_end are gcc builtins).